lib/data/multipleBackendGateway.js

```diff
@@ -77,9 +147,36 @@ const multipleBackendGateway = {
                     }
                     cb();
                 });
+            } else if (client.clientType === 'aws_s3') {
+                client.headBucket({ Bucket: client.awsBucketName },
+                err => {
+                    if (err) {
+                        multBackendResp[location] = { error: err };
```

Comment:
_Suggestion_ : adding a text before the error may help the user to figure out what is wrong, like you did for versioning. (AWS S3 bucket is missing)